src/Pages/Main/Components/BestItem/BestItem.scss

```
@@ -0,0 +1,68 @@
+@import '/src/Styles/variables.scss';
+
+.BestItem {
+  display: flex;
+  // padding-top 은 nav 머지 후 해제하겠습니다.
+  padding-top: 700px;
+  max-width: 1127px;
+  margin: 0 auto;
+  color: #333;
+  font-size: $main-font-size;
+
+  .sectionInfo {
+    min-width: 228px;
+
+    .title {
+      margin-bottom: 40px;
+      font-size: 32px;
+      font-weight: bold;
+    }
+
+    .subtitle {
+      margin-bottom: 27px;
+      color: #838383;
+      cursor: pointer;
+    }
+
+    .more {
+      margin-bottom: 45px;
+      font-size: 14px;
+      color: #393939;
+      cursor: pointer;
+
+      &::after {
+        content: '';
+        position: relative;
+        margin-left: -7px;
+        top: 9px;
+        display: inline-block;
+        width: 40px;
+        height: 30px;
+        transform: rotate(270deg) scale(0.3);
+        background: url('http://localhost:3000/images/sprite.png') -735px -265px
```

```suggestion
        background: url('/images/sprite.png') -735px -265px
```
localhost 3000은 로컬 환경에서만 작동하고, 배포한 후 라든가 실행 환경이 변하면 이미지를 찾지 못하게 됩니다.